core/lib/zksync_core/src/eth_sender/eth_tx_manager.rs

```diff
@@ -276,6 +283,29 @@ impl EthTxManager {
         Ok(OperatorNonce { finalized, latest })
     }
 
+    async fn get_blobs_operator_nonce(
+        &self,
+        block_numbers: L1BlockNumbers,
+    ) -> Result<Option<OperatorNonce>, ETHSenderError> {
+        match &self.ethereum_gateway_blobs {
```

Comment:
nit: could we merge it / refactor with the code above (get_operator_nonce), by having a common subfuncion that accepts 'gateway' as parameter?